### PR TITLE
Firmware Image as an example

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]

**Review by aj-stein-nist — APPROVED** [review]

**Review by roywill — APPROVED** [review]

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ Existing transparency systems such as Certificate Transparency are instances of
 A Signed Statement is an identifiable and non-repudiable Statement made by an Issuer.
 The Issuer selects additional metadata and attaches a proof of endorsement (in most cases, a signature) using the identity key of the Issuer that binds the Statement and its metadata.
 Signed Statements can be made transparent by attaching a proof of Registration by a Transparency Service, in the form of a Receipt that countersigns the Signed Statement and witnesses its inclusion in the Append-only Log of a Transparency Service.
-By extension, the document may say an Artifact (a firmware binary) is transparent if it comes with one or more Transparent Statements from its author or owner, though the context should make it clear what type of Signed Statements is expected for a given Artifact.
+By extension, the document may say an Artifact (for example, a firmware binary) is transparent if it comes with one or more Transparent Statements from its author or owner, though the context should make it clear what type of Signed Statements is expected for a given Artifact.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:286]:
```suggestion
By extension, the Signed Statement may say an Artifact (for example, a firmware binary) is transparent if it comes with one or more Transparent Statements from its author or owner, though the context should make it clear what type of Signed Statements is expected for a given Artifact.
```

Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:286]: Found a nit along the sides.

**Review by henkbirkholz — APPROVED** [review]
changes "document" to "Signed Statement" (was a weird nit close to the suggested change)

**Pull request merged**